enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+│   ├── capabilities_v1alpha1_metric_crd.yaml
+│   ├── capabilities_v1alpha1_plan_crd.yaml
+│   └── capabilities_v1alpha1_tenant_crd.yaml
+├── metadata
+│   └── annotations.yaml
+├── mybundle.tar
+└── root
+    └── buildinfo
+        ├── Dockerfile-myoperator-rhel7-operator-metadata-2.8.2-5
+        └── content_manifests
+            └── myopertor-bundle-container-2.8.2-5.json
+```
+
+#### Using SDK bundle validate
+
+[SDK](https://github.com/operator-framework/operator-sdk) allows us to validate all bundles: 
+
+```
+ $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --output=json-alpha1
+
+INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
+{
+    "passed": false,
+    "outputs": [
+        {
+            "type": "error",
+            "message": "Error: Value : (memcached-operator.v0.0.1) csv.Spec.Icon elements should contain both data and mediatype"
+        }
+    ]
+}
+```
+
+Currently, the validators used by it will only ensure the bundle [spec definition](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md) and the standard requirements to publish an operator in the OperatorHub.io. For further information also check the proposal [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub].
+
+#### Using SDK scorecard bundle
+
+For the SDK tool be able to run scorecard tests its configuration requires to be wrote in the bundle directory `tests/scorecard/config.yaml`. ([example](https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/tests/scorecard/config.yaml)). Operator bundles which are build with SDK will have the scorecard tests configured by default. ([example](https://github.com/operator-framework/community-operators/tree/master/community-operators/namespace-configuration-operator/1.0.1)).
+ 
+Note that the bundles might have some specific tests implemented by them using the option to write custom tests. For further information check the [Writing Custom Scorecard Tests](https://sdk.operatorframework.io/docs/advanced-topics/scorecard/custom-tests/) documentation. In this way, if a bundle does not have the default scorecard test configured audit command will add it. Otherwise, the audit command should ensure that specific configuration will be used instead. 
+
+**NOTE** It is out of the scope of this proposal, however, SDK could allow this configuration also be performed via the plugin system such as by let us execute the plugin which is responsible for that, e.g  `operator-sdk run --plugins=scorecard.sdk.operatorframework.io/v2`, instead of the audit command requires re-implement it. See that it might be easily achievable after the [plugin systems be improved](https://github.com/kubernetes-sigs/kubebuilder/issues/2016).
```

Comment:
This is a confusing aside since no `operator-sdk run` command exists, and the description is vague and is not in context here. I'd like to see this removed.
```suggestion
```

Reply:
The note is to illustrate and bring context for the others that in the future would be possible to have a helper in SDK via a plugin that would scaffold the manifests required for we add the scorecard config in the bundle when it is not found. Also, for who will implement it to know how to handle the code and add comments to make it easier we might replace that in the future for an SDK helper.

Reply:
Removed and better explained in the risks.